docs/user-guide/connectors/slack.rst

```diff
@@ -17,10 +17,16 @@ Getting Credentials
 
   1. To create the app go to: https://api.slack.com/apps and click
      on *"Create New App"*.
-  2. Activate the following features: interactive components, event
-     subscriptions, bot users, permissions (for basic functionality
-     you should subscribe to the ``message.channel``,
-     ``message.groups``, ``message.im`` and ``message.mpim`` events)
+  2. Activate the following features: 
```

Comment:
I think you need another line after this one in order to get the list to render properly